Firestore/Example/Tests/Local/FSTLocalStoreTests.mm

```diff
@@ -249,8 +250,12 @@ - (void)testHandlesSetMutationThenDocument {
   FSTAssertChanged(@[ FSTTestDoc("foo/bar", 0, @{@"foo" : @"bar"}, YES) ]);
   FSTAssertContains(FSTTestDoc("foo/bar", 0, @{@"foo" : @"bar"}, YES));
 
-  [self applyRemoteEvent:FSTTestUpdateRemoteEvent(
-                             FSTTestDoc("foo/bar", 2, @{@"it" : @"changed"}, NO), @[ @1 ], @[])];
+  FSTQuery *query = FSTTestQuery("foo");
```

Comment:
Few of these queries seem to actually matter. Would it be worth factoring this dance out into a separate method like this?

```objc
- (void)applyRemoteEventForDocument:(FSTDocument *)document {
  FSTQuery *query = FSTTestQuery("foo");
  FSTTargetID targetID = [self allocateQuery:query];
  [self applyRemoteEvent:FSTTestUpdateRemoteEvent(document, @[ @(targetID) ], @[])];
  [self.localStore releaseQuery:query];
}
```

Just to cut down on the noise in the tests where they don't care?

Reply:
I see. Carry on then :-).